### ISSUE TITLE
socket.forEach is not a function lib/server.js

### ISSUE THREAD

**Issue by NirBenita** [karma-runner/karma#1784]
```
Missing error handler on `socket`.
TypeError: sockets.forEach is not a function
    at disconnectBrowsers (.../node_modules/karma/lib/server.js:314:13)
    at null.<anonymous> (.../node_modules/karma/lib/server.js:291:7)
    at emitTwo (events.js:92:20)
    at emit (events.js:172:7)
    at emitRunCompleteIfAllBrowsersDone (.../node_modules/karma/lib/server.js:256:12)
    at null.<anonymous> (.../node_modules/karma/lib/server.js:278:9)
    at emitTwo (events.js:92:20)
    at emit (events.js:172:7)
    at onComplete (.../node_modules/karma/lib/browser.js:142:13)
    at Socket.<anonymous> (.../node_modules/karma/lib/events.js:13:22)
    at emitTwo (events.js:92:20)
    at Socket.emit (events.js:172:7)
```

**Comment by Dignifiedquire** [comment]
Fixed in 0.13.19

**Comment by NirBenita** [comment]
@Dignifiedquire thanks, i was actually about to say that :+1:
